modules/lang-painless/src/main/java/org/elasticsearch/painless/Utility.java

```diff
@@ -248,6 +248,10 @@ public static Double charToDouble(final char value) {
         return (double)value;
     }
 
+    public static String charToString(final char value) {
+        return "" + value;
+    }
+
```

Comment:
Can we do this one as `String.valueOf`? https://docs.oracle.com/javase/7/docs/api/java/lang/String.html#valueOf%28char%29